api/client/cluster/client.go

```diff
@@ -142,94 +142,6 @@ func (c *clusterClient) EnableUpdates() error {
 	return nil
 }
 
-func (c *clusterClient) SetLoggingURL(loggingURL string) error {
```

Comment:
Mark it as deprecated. And we will define a strategy to deprecate this if not needed.